### DISCUSSION TITLE
Task 1

### DISCUSSION THREAD

**Discussion by irsyaad06** [KOSATAN7/embrioclass#16] [Announcements]
![task 1](https://github.com/KOSATAN7/embrioclass/assets/86083982/31343ac9-0758-4ffc-89e7-6b23921c5dec)

Tolong buat Seperti itu dengan menggunakan Tailwind 

Link Documen tati on : 
-> https://tailwindcss.com/docs/installation
-> Pake Tailwind via CDN #NGGA ADA INSTALL KADE

IQRA THE DOCUMENTATION! 🥱
Clue :
-> gaada clue, baca sorangan 🙄

**Comment by iqbaljnr** [comment]


**Reply by iqbaljnr** [reply]
akuu masukin gambar aku di src ini? aku bikin folder lagi ga

**Reply by Rifqi1012** [reply]
Gausah kan udah disediain

**Reply by Rifqi1012** [reply]
Di src/images/nama kamu
